services/api/src/unified_graphics/routes.py

```diff
@@ -22,15 +22,13 @@ def index():
     return jsonify({"msg": "Hello, Dave"})
 
 
-@bp.route("/diag/wind/<loop>/<value_type>/")
-def wind(loop, value_type):
-    try:
-        loop_enum = diag.MinimLoop(loop)
-        type_enum = diag.ValueType(value_type)
-    except ValueError:
-        raise FileNotFoundError()
-
-    return jsonify(diag.wind(loop_enum, type_enum))
+@bp.route("/diag/wind/")
```

Comment:
Just checking - this is hardcoded to the `uv` wind variable/files for now, correct?

Reply:
That’s correct. For the moment, we have no other variables that are vectors, so there hasn’t been a need to refactor this and update these routes. I suspect that #74 will be an impetus to refactor the routes to be more general, since we’ll be adding more scalar variables.